### PR TITLE
Draw a UML Class Diagram

### PR DESCRIPTION
**Pull request by EJEunjiyaz**

### CODE REVIEW

File: draw-uml/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Draw a UML Class Diagram
+>The Unified Modeling Language (UML) is a general-purpose, developmental, modeling language in the field of software engineering that is intended to provide a standard way to visualize the design of a system.
+
+from [Wikipedia](https://en.wikipedia.org/wiki/Unified_Modeling_Language)
+
+## Question
+
+Given `Animal` as an object. This object have `name(string)`, `id(int)`, `age(int)`, and `weight(int)` as attributes. It has 3 different kinds of animals. The first one is `Tortoise`, the second is `Cat`, and the last one is `Bird`. These 3 animals are the child class of the `Animal`. All these 3 animals have the same attributes as the `Animal` except for the `Cat` has one more attribute named `leg_length(int)`. However, if two or more `Tortoise` live together, then they can be called the `Creep`. The `Animal` also have function `eat()`. Only `Cat` can eat the `Bird`.
```
Comment (jbrucker) [inline draw-uml/README.md]: object have -> object has
"It has" -> "There are"
"These 3 animals are the child class..." -> delete.  Let students figure it out themselves.

"can be called the Creep"  do you mean "they form a group called a Creep"

File: draw-uml/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Draw a UML Class Diagram, show the relationship of each object by using symbols or the link.
```
Comment (jbrucker) [inline draw-uml/README.md]: "of each object by using symbols or the link"  What "link"?

should be : "between objects using UML notation".

File: draw-uml/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## Answer
+
+![Screenshot](https://github.com/ISP19/problems/blob/vichyawat/draw-uml/Class%20Diagram%20with%20UML%20Notation.png)
+
+- Use ⇾ for `Tortoise`, `Cat` and `Bird` to show the **inheritance** that there is the subclass of `Animal`.
+
+- Use ━◇ for `Creep` to show the **aggregation** that `Tortoise` is part of the `Creep`
+
+- Use ━ for `Cat` and `Bird` to show the **association** that they can eat or be eaten.
```
Comment (jbrucker) [inline draw-uml/README.md]: Does Cat have an attribute of type Bird?

If the cat merely references the bird class then it is dependency, not association
```
def can_eat( food: Animal ):
    if isinstance(food, Bird) or isinstance(food, Fish):
        return true
    return false
```

Comment (EJEunjiyaz) [reply draw-uml/README.md]: Yes, it's not an association. Thank you.

**Comment by EJEunjiyaz** [comment]
I fixed it, please check again.

Thank you.

**Pull request merged**